core/src/main/scala/org/ensime/core/javac/JavaCompletion.scala

```diff
@@ -209,7 +210,7 @@ trait JavaCompletion extends Helpers with SLF4JLogging {
     caseSense: Boolean
   ): List[CompletionInfo] = {
 
-    typeElement(info, target).map {
+    typeElement(info, target).toList.flatMap {
```

Comment:
I must admit, there is something here that I am missing: so you're saying that this: `Option(1).flatMap(x => List(x))` should compile, right?